compiler/lib/src/main/scala/codegen/CppWriter/AliasCppWriter.scala

```diff
@@ -0,0 +1,185 @@
+    s: CppWriterState,
+    aNode: Ast.Annotated[AstNode[Ast.DefAliasType]]
+  ): List[String] = {
+    val Right(a) = UsedSymbols.defAliasTypeAnnotatedNode(s.a, aNode)
+    s.writeIncludeDirectives(a.usedSymbolSet)
+  }
+
+  private def writeHIncludeDirectives(
+    s: CppWriterState,
+    aNode: Ast.Annotated[AstNode[Ast.DefAliasType]]
+  ): List[String] = {
+    val Right(a) = UsedSymbols.defAliasTypeAnnotatedNode(s.a, aNode)
+    def getIncludeFiles(sym: Symbol): Option[String] = {
+      val name = s.getName(sym)
+      for {
+        fileName <- sym match {
+          case Symbol.AliasType(_) => Some(ComputeCppFiles.FileNames.getAliasType(name))
+          case Symbol.AbsType(node) =>
+             if s.isBuiltInType(name) then Some("FppConfig") else None
+          case _ => None
+        }
+      }
+      yield s.getIncludePath(sym, fileName, "h")
+    }
+
+    a.usedSymbolSet.map(getIncludeFiles).filter(_.isDefined).map(_.get).map(CppWriterState.headerString).toList
+  }
+
+  def writeHpp: CppDoc = {
+    val includeGuard = s.includeGuardFromQualifiedName(symbol, fileName, "HPP")
+    CppWriter.createCppDoc(
+      s"$name alias",
+      fileName,
+      includeGuard,
+      getHppMembers,
+      s.toolName,
+      "hpp"
+    )
+  }
+
+  def writeH: Option[CppDoc] = {
+    if (!supportedCHeader) {
+      return None
+    }
+
+    val includeGuard = s.includeGuardFromQualifiedName(symbol, fileName, "H")
+    Some(CppWriter.createCppDoc(
+      s"$name alias",
+      fileName,
+      includeGuard,
+      getHMembers,
+      s.toolName,
+      "h"
+    ))
+  }
+
+  private def getHppMembers: List[CppDoc.Member] = {
+    List.concat(
+        List(getHppIncludes),
+        wrapInNamespaces(namespaceIdentList, List(getHppDefinition))
+    )
+  }
+
+  private def getHMembers: List[CppDoc.Member] = {
+    List(
+      getHIncludes,
+      getHDefinition,
+    )
+  }
+
+  private def getHPPCPPGuard: CppDoc.Member = {
+    linesMember(addBlankPrefix(lines(
+        """extern "C" {""")
+      ))
+  }
+
+  private def getHPPCloseCPPGuard: CppDoc.Member = {
+    linesMember(addBlankPrefix(lines(
+        """}""")
+      ))
+  }
+
+  private def getHIncludes: CppDoc.Member = {
+    if (!supportedCHeader) {
+      // C header is not supported, we will generate the definition
+      // in the C++ header
+      return linesMember(List())
+    }
+
+    val standardHeaders = List(
+      "Fw/Types/BasicTypes.h",
+    ).map(CppWriter.headerString)
+    val symbolHeaders = writeHIncludeDirectives(s, aNode)
+    val headers = standardHeaders ++ symbolHeaders
+    linesMember(addBlankPrefix(headers.distinct.sorted.map(line)))
+  }
+
+  private def getHppIncludes: CppDoc.Member.Lines = {
+    val standardHeaders = List(
+      aliasType.aliasType match {
+        case Type.String(_) => "Fw/Types/String.hpp"
+        case Type.AbsType(node) =>
+           s.isBuiltInType(node._2._1.name) match {
+            case true => "FppConfig.hpp"
```

Comment:
Let's make FpConfig.hpp a system header. Also I think it's FpConfig.hpp and not FppConfig.hpp.

Reply:
It seems we need to add a test for this case.